### DISCUSSION TITLE
KEDA v2.14.0

### DISCUSSION THREAD

**Discussion by zroubalik** [kedacore/keda#5748] [Announcements]
We are happy to release KEDA v2.14.0 🎉

Here are some highlights:
- Provide capability to filter CloudEvents (https://github.com/kedacore/keda/issues/3533)
- (Experimental) Introduce Azure Event Grid as a new CloudEvent destination (https://github.com/kedacore/keda/issues/3587)
- Support Kafka SASL MSK IAM authentication
- Multiple OpenTelemetry improvements
- Multiple ScalingModifiers improvements and fixes

Here are the new deprecation(s) as of this release:

- Various Prometheus metrics have been renamed to follow the preferred naming conventions. The old ones are still available, but will be removed in the future (https://github.com/kedacore/keda/issues/4854).

Learn how to deploy KEDA by reading [our documentation](https://keda.sh/docs/2.14/deploy/).

🗓️ The next KEDA release is currently being estimated for July 9th, 2024, learn more in our [roadmap](https://github.com/kedacore/keda/blob/main/ROADMAP.md#upcoming-release-cycles).

## New Contributors
* @rahulii made their first contribution in https://github.com/kedacore/keda/pull/5446
* @vinod827 made their first contribution in https://github.com/kedacore/keda/pull/5518
* @junekhan made their first contribution in https://github.com/kedacore/keda/pull/5481
* @atilsensalduz made their first contribution in https://github.com/kedacore/keda/pull/5433
* @pauldotyu made their first contribution in https://github.com/kedacore/keda/pull/5517
* @ArunYogesh made their first contribution in https://github.com/kedacore/keda/pull/5483
* @tico88612 made their first contribution in https://github.com/kedacore/keda/pull/5566
* @cyrilico made their first contribution in https://github.com/kedacore/keda/pull/5621
* @msvticket made their first contribution in https://github.com/kedacore/keda/pull/5592
* @binboum made their first contribution in https://github.com/kedacore/keda/pull/5634
* @windy1 made their first contribution in https://github.com/kedacore/keda/pull/5600
* @nicholass-alcidion made their first contribution in https://github.com/kedacore/keda/pull/5672
* @jkremser made their first contribution in https://github.com/kedacore/keda/pull/5631
* @goodfirm made their first contribution in https://github.com/kedacore/keda/pull/5673
* @xrwang8 made their first contribution in https://github.com/kedacore/keda/pull/5478
* @jtymes made their first contribution in https://github.com/kedacore/keda/pull/5701
* @guopeng0 made their first contribution in https://github.com/kedacore/keda/pull/5629
* @adrien-f made their first contribution in https://github.com/kedacore/keda/pull/5692

**Full Changelog**: https://github.com/kedacore/keda/compare/v2.13.0...v2.14.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/keda/releases/tag/v2.14.0'>KEDA v2.14.0</a>.</em>

**Comment by qpetraroia** [comment]
@zroubalik Does KEDA 2.14 support k8s 1.30?
